recipes/cython/osx-post-link.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $PREFIX/lib
+rm -f libgcc_s.10.5.dylib
+ln -s /usr/lib/libgcc_s.1.dylib libgcc_s.10.5.dylib
```

Comment:
Well at least you have a decent test case to see if this is an issue. With any luck, this is an ancient workaround for an issue already fixed by upstream.